runtime/src/account_storage.rs

```diff
@@ -48,6 +48,14 @@ impl AccountStorage {
             .unwrap_or(true)
     }
 
+    /// returns true if there is an entry in the map for 'slot', but it contains no append vec
+    #[cfg(test)]
```

Comment:
minor: consider putting this with the other `#[cfg(test)]` fns (at the bottom of the impl block)